echo/frontend/src/routes/Debug.tsx

```
@@ -738,6 +738,7 @@ export default function DebugPage() {
 
   const { data: conversation } = useConversationById({
     conversationId: currentConversationId!,
+    loadConversationChunks: true,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)